packages/terra-clinical-detail-view/src/DetailView.jsx

```diff
@@ -56,10 +74,18 @@ const DetailView = ({ title, subtitles, graph, details, footer, isDivided, ...cu
   let divider = null;
```

Comment:
Nit: for readability, lets move `divider` and `dividedDetails`  to before they are used, on line 90.